scripts/flask/web_interface.py

```
@@ -379,7 +393,7 @@
 </html>
 """
 
-def run_script_with_live_output(script_name):
+def run_script_with_live_output(script_name, arg_values=[]):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)